py/serve.py

```diff
@@ -970,3 +968,8 @@ def logout():
     response = flask.redirect('/login')
     response.set_cookie('Access-Token', '', expires=0)
     return response
+
+
+def serialize_for_db(dataset):
+    """Serializes a Dataset for input into postgres."""
+    return dataset.SerializeToString().hex()
```

Comment:
`deterministic=True`?

Reply:
Doesn't really matter here (since we're not versioning against the exported bytes), but sure.